Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Hotfix for BTS report #684393:
Don't crash on quotation marks in titles of Norwegian Nynorsk documents.
--- a/latex/style/dbk_locale.sty
+++ b/latex/style/dbk_locale.sty
@@ -268,6 +268,10 @@
   \ifxetex\usepackage{xecyr}\fi
 }
 
+\babelsetup{nn}{%
+  \AtBeginDocument{\shorthandoff{"}}
+}
+
 %% Apply default locale setup
 \setuplocale{en}
 
